docs/feature-facades.rst

```diff
@@ -167,4 +167,4 @@ showing how this is done:
     }
 
 Alternatively, if you are specifically only mocking responses from clients, you might consider using the `Guzzle Mock
-Plugin <http://guzzlephp.org/plugins/mock-plugin.html>`_.
+Plugin <https://guzzle3.readthedocs.io/plugins/mock-plugin.html>`_.
```

Comment:
This is not the correct link.. this is from Guzzle v3 which is not supported anymore...

This is the link for G6-7: http://docs.guzzlephp.org/en/stable/testing.html#mock-handler
And this is the link fro G5: http://docs.guzzlephp.org/en/5.3/testing.html#mock-subscriber

Unless if the doc here is intent is to point to Guzzle 3

Reply:
This change is for the documentation for version 2 of the AWS SDK for PHP, which depends on Guzzle 3. That said, I suppose there's no reason not to include links for mocking HTTP responses using newer versions of Guzzle.